optimum/commands/export/openvino.py

```diff
@@ -274,6 +274,13 @@ def parse_args_openvino(parser: "ArgumentParser"):
             "reduces quantization error. Valid only when activations quantization is enabled."
         ),
     )
+    optional_group.add_argument(
+        "--eagle3",
+        action="store_true",
+        help=(
+            "the original model is a draft model of eagle3 pipeline."
```

Comment:
You require to run `optimum-cli` with `--task text-generation-with-past`. Is it possible to detect it automatically? Same for  `--eagle3`